src/main/java/winstone/HostConfiguration.java

```diff
@@ -98,11 +94,12 @@ public HostConfiguration(Server server, String hostname, ClassLoader commonLibCL
                     }
                     String extension = mapping.substring(0, delimPos);
                     String mimeType = mapping.substring(delimPos + 1);
-                    this.mimeTypes.addMimeMapping(extension.toLowerCase(), mimeType);
+                    server.getMimeTypes().addMimeMapping(extension.toLowerCase(), mimeType);
                 }
             }
         }
-
+        Optional.ofNullable(configureAccessLog("webapp")).ifPresent(server::setRequestLog);
+        WebAppContext handler = create(getWebRoot(webroot, warfile), prefix);
```

Comment:
Why move this from before loading the MIME types to after? Seems like the diff would be easier to read if various parts of the code weren't unnecessarily moved.